pipelines/build-testapp.yml

```diff
@@ -94,9 +94,256 @@ stages:
         targetPath: $(artifactDir)
         artifactName: $(artifactName)
 
+  - job: test_app_android
+    timeoutInMinutes: 180
+    displayName: 'test app Android'
```

Comment:
Maybe 'Build Android Test App' is better?

Reply:
updated